core/src/main/java/com/google/cloud/sql/core/DefaultAccessTokenSupplier.java

```diff
@@ -50,7 +48,7 @@ class DefaultAccessTokenSupplier implements AccessTokenSupplier {
    *
    * @param tokenSource the token source that produces auth tokens.
    */
-  DefaultAccessTokenSupplier(HttpRequestInitializer tokenSource) {
+  DefaultAccessTokenSupplier(CredentialFactory tokenSource) {
```

Comment:
Replace `HttpRequestInitializer` with `CredentialFactory`. The AccessTokenSupplier produces access tokens using a GoogleCredential. It shouldn't need to know about an HttpRequestInitializer.